### DISCUSSION TITLE
Nombre de caractéristiques disponibles

### DISCUSSION THREAD

**Discussion by ElethornGM** [fibreville/minotaure_php#61] [Boite à idées]
Actuellement il y a deux caractéristiques (esprit/corps par défaut, modifiables) et une statistique supplémentaire (Points de vie).
Une idée serait de revoir ça à la hausse, ou de pouvoir définir le nombre de caractéristiques dans les paramètres de la partie.

Avoir jusqu'à 3-5 caractéristiques pourrait apporter plus de différenciation et d’immersion pour les joueurs.

Avoir une seconde statistique (mana, énergie, ...) et pouvoir la nommer (par exemple "argent" ou "pouvoir"...) pourrait être un plus.

**Comment by uurshin** [comment]
C'est une assez grosse modif, mais faisable. Par contre faudrait réfléchir en amont à la place que ça va prendre sur l'écran du MJ d'avoir X PJs avec Y caracs.

Très bonne idée pour la seconde stat (enfin du coup on limitera probablement pas à deux, autant y aller à fond) !

**Reply by ElethornGM** [reply]
Potentiellement, une solution serait que les caractéristiques ne soient pas display sur l'écran de MJ, mais quand tu passes ta souris sur la fiche d'un PJ (via l'écran de MJ) il y a un genre de pop-up plus détaillé avec les infos des caractéristiques. T'en penses quoi ?

**Reply by uurshin** [reply]
Je viens de penser qu'avec du D20, ça n'aura plus vraiment autant de sens d'afficher uniquement les "forts" dans une carac, il faudra avoir 2 adjectifs par carac. Exemple :
- force inférieure à 6 = "faible"
- force supérieure à 14 = "fort"

On pourrait alors se content d'afficher ces adjectifs au lieu des caracs, et omettre complètement l'information pour les personnages qui sont "moyens".
Vu que la création de personnage (cf #62) fera en sorte que le total des carac soit le même pour tout le monde, ça devrait naturellement équilibrer le nombre de lignes où un adjectif est affiché et éviter d'avoir une ligne affiché par carac.

Ca donnerait, pour trois caracs (force, intelligence, charisme) :

Brad : Malin (18), Repoussant (3)
Cindy : Fort (15)
Artyom : Stupide (4), Avenant (18) 

En survolant le personnage on pourrait avoir le détail (qui n'est pas méga intéressant pour l'aventure).

**Comment by ElethornGM** [comment]
Une idée comme ça, si on augmente le nombre de caractéristiques et qu'on les cache (et qu'on les voit en survolant le personnage), ça pourrait être bien d'augmenter la taille du nom de chaque personnage (grosseur des caractères). J'ai eu plusieurs retours en stream qu'ils avaient du mal à lire l'écran (vu que la plupart du temps ils ne se mettent pas en plein écran pour pouvoir parler dans le tchat).

**Reply by uurshin** [reply]
Par rapport à ta dernière remarque, je pense qu'avec la dernière version ça pose moins problème ? Vu qu'on peut mettre la grille des joueurs sur + de largeur maintenant. Dans tous les cas j'ai légèrement augmenté la taille de tout le texte des cases de personnages. Je verrai pour faire un truc évolutif selon la taille du public (+ petit si plus de 100 joueurs par exemple).

**Reply by ElethornGM** [reply]
Ouai c'est mieux avec le nouveau display je pense
